src/jit/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright 2017 wasmjit-omr project participants
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+add_library(wabtjit STATIC
+    type-dictionary.cpp
+    function-builder.cpp
+    wabtjit.cpp
+)
+
+target_link_libraries(wabtjit jitbuilder)
```

Comment:
If we're going to link jitbuilder here, this should probably be marked as `PUBLIC` and the `target_link_libraries` call for jitbuilder being linked into libwabt should be removed.

This also causes a regression on #6, since jitbuilder include files are not set as system include files in the context of JIT source files. Please adapt the code that solved that issue (see 4bee644c466fdc8e992de3d62e63e5b4358fdf5b) to work on the wabtjit target.